docusaurus/sidebars.js

```diff
@@ -530,10 +530,16 @@ const sidebars = {
       collapsed: false,
       label: 'Getting Started',
       items: [
+        'cloud/getting-started/intro',
         {
-          type: 'autogenerated',
-          dirName: 'cloud/getting-started'
-        }
+          type: 'doc',
+          id: 'cloud/getting-started/deployment',
+          customProps: {
+            updated: true,
```

Comment:
This is just to add an "Updated" badge to the table of content:
![Screenshot 2023-08-22 at 14 26 45](https://github.com/strapi/documentation/assets/4233866/90bcb9d0-a0ea-4cb0-ba5f-007141f74ea0)